#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
PodHeader pod_header
int8 manual_steer_angle
uint8 inboard_eps
uint8 outboard_eps
uint8 inboard_eps_error
uint8 outboard_eps_error
uint8 brake_pedal_force